config/sync/core.extension.yml

```diff
@@ -141,6 +141,7 @@ module:
   origins_shamrock: 0
   origins_shamrock_admin: 0
   origins_social_sharing: 0
+  origins_term_delete: 0
```

Comment:
This will work when https://github.com/dof-dss/nicsdru_origins_modules/pull/139/ is merged and composer is able to fetch either the development branch, or a tagged release containing it.